rmf_traffic/test/unit/agv/test_Planner.cpp

```diff
@@ -458,6 +458,159 @@ SCENARIO("Test Options", "[options]")
     CHECK(*default_options.interrupt_flag());
   }
 
+  Planner::Options default_maxcostestimate(nullptr);
+  WHEN("Maximum cost estimate is default after default construction")
+  {
+    // Converting an optional to a bool returns false if it's empty, which is
+    // the default for constructing Options
+    CHECK(!default_maxcostestimate.maximum_cost_estimate());
+  }
+
+  double local_maximum_cost_estimate(3.14159);
```

Comment:
```suggestion
  double local_maximum_cost_estimate = 3.14159;
```

I'd recommend the assignment approach to initialization since I think that's usually more clear.